### ISSUE TITLE
Send Eric the exact DSL commands for demo

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#673]
So he can copy-paste them into the iPython demo examples.

**Comment by dkoslicki** [comment]
Use ARAX_query.py 1212 for demo 2

**Comment by dkoslicki** [comment]
@edeutsch Waiting until I can finish the instructions from #663 before I can test and be sure the planned demo examples still work. Hopefully will get these to you my tomorrow morning

**Comment by dkoslicki** [comment]
**Demo example 1:**
```python
query = {"previous_message_processing_plan": {"processing_actions": [
            "create_message",
            "add_qnode(name=acetaminophen, id=n0)",
            "add_qnode(type=protein, id=n1)",
            "add_qedge(source_id=n0, target_id=n1, id=e0)",
            "expand(edge_id=e0)",
            "resultify(ignore_edge_direction=true)",
            "filter_results(action=limit_number_of_results, max_results=10)",
            "return(message=true, store=false)",
        ]}}
```
Runs just fine.

**Demo example 2:**
```python
query = { "previous_message_processing_plan": { "processing_actions": [
            "create_message",
            "add_qnode(curie=DOID:14330, id=n00)",
            "add_qnode(type=protein, is_set=true, id=n01)",
            "add_qnode(type=chemical_substance, id=n02)",
            "add_qedge(source_id=n00, target_id=n01, id=e00)",
            "add_qedge(source_id=n01, target_id=n02, id=e01, type=physically_interacts_with)",
            "expand(edge_id=[e00,e01], kp=ARAX/KG1)",
            "overlay(action=compute_jaccard, start_node_id=n00, intermediate_node_id=n01, end_node_id=n02, virtual_edge_type=J1)",
            "filter_kg(action=remove_edges_by_attribute, edge_attribute=jaccard_index, direction=below, threshold=.2, remove_connected_nodes=t, qnode_id=n02)",
            "filter_kg(action=remove_edges_by_property, edge_property=provided_by, property_value=Pharos)",
            "overlay(action=predict_drug_treats_disease, source_qnode_id=n02, target_qnode_id=n00, virtual_edge_type=P1)", 
            "resultify(ignore_edge_direction=true)",
            "filter_results(action=sort_by_edge_attribute, edge_attribute=probability_drug_treats, direction=descending, max_results=15)", 
            "return(message=true, store=false)",
            ] } }
```
`resultify()` is throwing an error here. See #676. This one was working just fine before recent changes to `resultify()` by @saramsey 

**Demo example 3:**
```python
query = {"previous_message_processing_plan": {"processing_actions": [
            "create_message",
            "add_qnode(curie=DOID:9406, id=n00)",
            "add_qnode(type=chemical_substance, is_set=true, id=n01)",
            "add_qnode(type=protein, id=n02)",
            "add_qedge(source_id=n00, target_id=n01, id=e00)",  
            "add_qedge(source_id=n01, target_id=n02, id=e01)",
            "expand(edge_id=[e00,e01])", 
            "overlay(action=overlay_clinical_info, observed_expected_ratio=true, virtual_edge_type=C1, source_qnode_id=n00, target_qnode_id=n01)", 
            "filter_kg(action=remove_edges_by_attribute, edge_attribute=observed_expected_ratio, direction=below, threshold=3, remove_connected_nodes=t, qnode_id=n01)",   
            "filter_kg(action=remove_orphaned_nodes, node_type=protein)",
            "overlay(action=compute_ngd, virtual_edge_type=N1, source_qnode_id=n01, target_qnode_id=n02)", 
            "filter_kg(action=remove_edges_by_attribute, edge_attribute=ngd, direction=above, threshold=0.85, remove_connected_nodes=t, qnode_id=n02)",
            "resultify(ignore_edge_direction=true)",
            "return(message=true, store=false)"
        ]}}
```
Runs just fine

**Comment by edeutsch** [comment]
I have committed and pushed updated to the Jupyter Notebooks for examples 1,2,3. All seem to work function well, although output may still need refinement.

Editorial changes that I made:
1) I removed "create_message" from all of them since this is not necessary. It is assumed that if you do not provide an existing input message, the first step is to create a blank message. In my push to simplify simply, I took it out.
2) I changed store=true for all these. For internal testing, it's fine to use store=false so as not to fill the database with lots of tests. But for the API Notebooks, things the experience is degraded if you don't store, because then there's id to reference.

I haven't done it yet, but I'm planning to make message=true, store=false the defaults and have it so that if you do not provide an explicit return(), one is implied and will happen with those defaults. Again, to simplify. Let me know if you object!
